pyo3-ffi/src/cpython/mod.rs

```diff
@@ -58,6 +60,7 @@ pub use self::object::*;
 pub use self::objimpl::*;
 pub use self::pydebug::*;
 pub use self::pyerrors::*;
+pub use self::pyframe::*;
```

Comment:
```suggestion
#[cfg(not(PyPy))]
pub use self::pyframe::*;
```

You need it on the re-export as well, since the module is not available on PyPy